regex-parsing/src/test/java/org/sonarsource/analyzer/commons/regex/finders/RegexFinderVerifier.java

```diff
@@ -168,16 +168,24 @@ public void reportInvocationTreeIssue(Node regexNode, String message, @Nullable
     }
 
     private void reportIssue(IssueLocation.Range range, String message, List<RegexIssueLocation> secondaries) {
+      verifyHighlightableRange(range);
       SingleFileVerifier.Issue issue = verifier
         .reportIssue(message)
         .onRange(range.getLine(), range.getColumn(), range.getEndLine(), range.getEndColumn());
       secondaries.forEach(location -> {
         IssueLocation.Range secondaryRange = issueLocationInFile(location);
+        verifyHighlightableRange(range);
         issue.addSecondary(secondaryRange.getLine(), secondaryRange.getColumn(), secondaryRange.getEndLine(),
           secondaryRange.getEndColumn(), location.message());
       });
     }
 
+    private void verifyHighlightableRange(IssueLocation.Range range) {
+      if (range.getLine() == range.getEndLine() && range.getColumn() == range.getEndColumn()) {
```

Comment:
I am not sure I understand what exactly this condition means.

Reply:
The parser creates trees, especially for an empty element which have a range of 0. In other words, the range of a tree could be from column 11 to column 11. This is a range inside the code that can not be highlighted aka displayed in the UI.